test-case/check-sof-logger.sh

```diff
@@ -233,14 +233,20 @@ reload_drivers()
 
     # The DSP may unfortunately need multiple retries to boot, see
     # https://github.com/thesofproject/sof/issues/3395
+    # Timeout needs to be 60+sec as platform with GFX present
+    # on hardware but without functional i915 driver, the SOF
+    # probe will require the 60sec i915 probe timeout to expire
+    # before card probe can be completed.
     dlogi "Polling ${CARD_NODE}, waiting for DSP boot..."
-    for i in $(seq 1 5); do
+    for i in $(seq 1 70); do
```

Comment:
Can you use `MAX_WAIT_FW_LOADING` here? See #1059 for other use cases

Reply:
If you look at the (small) PRs linked from #1059 you will see a new, convenience function `poll_wait_for()` function that can likely replace this loop.